### PR TITLE
read me

### PR DESCRIPTION
**Pull request by bnaly16**
tell me details

### CODE REVIEW

**Comment by bnaly16** [comment]
ok

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # cli_project
 local command line project
+
+مش فاهم حاااااااااااااجااااااااااااااة
```
Comment (MuhammadBnAly) [inline README.md:4]: repeat this code, not correct

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 # cli_project
 local command line project
+
```
Comment (MuhammadBnAly) [inline README.md:3]: hello world

**Review by MuhammadBnAly — COMMENTED** [review]
allah akbar

**Pull request merged**